### PR TITLE
today i make changes in renewal services

### PR DESCRIPTION
**Pull request by Damodar86**

### CODE REVIEW

File: src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java [hunk 1]
```diff
@@ -23,11 +24,25 @@ public BookRenewalService(BookCheckOutRepository bookCheckOutRepository) {
     public String bookRenewalServiceMethod(BookRenewalModel bookrenewalmodel) throws LibraryException {
         Optional<BookCheckOutEntity> bookCheckOutEntity = bookCheckOutRepository.findByBookBarCodeAndBorrowerAccountIdAndReturnedDateIsNull(bookrenewalmodel.getBarCode(), bookrenewalmodel.getAccountId());
         if (bookCheckOutEntity.isPresent()) {
-            return null;
+            log.info("entered into bookCheckOutEntity is Present block");
+            BookCheckOutEntity bookCheckOutEntity1 = bookCheckOutEntity.get();
```
Comment (user554) [inline src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java:28]: looks good

File: src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java [hunk 2]
```diff
@@ -23,11 +24,25 @@ public BookRenewalService(BookCheckOutRepository bookCheckOutRepository) {
     public String bookRenewalServiceMethod(BookRenewalModel bookrenewalmodel) throws LibraryException {
         Optional<BookCheckOutEntity> bookCheckOutEntity = bookCheckOutRepository.findByBookBarCodeAndBorrowerAccountIdAndReturnedDateIsNull(bookrenewalmodel.getBarCode(), bookrenewalmodel.getAccountId());
         if (bookCheckOutEntity.isPresent()) {
-            return null;
+            log.info("entered into bookCheckOutEntity is Present block");
```
Comment (user554) [inline src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java:27]: Add more logging details

Comment (Damodar86) [reply src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java:27]: done

File: src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java (hunk 1 above)
Comment (Damodar86) [reply src/main/java/com/libraryManagement/libraryArtifact/libraryService/BookRenewalService.java:28]: done

**Pull request merged**